domain/life/life.go

```diff
@@ -12,3 +14,17 @@ const (
 	Dying
 	Dead
 )
+
+// Value returns the [github.com/juju/juju/core/life.Life]
+// value corresponding to this life.
+func (l Life) Value() corelife.Value {
+	switch l {
+	case Alive:
+		return corelife.Alive
+	case Dying:
+		return corelife.Dying
+	case Dead:
+		return corelife.Dead
+	}
+	return corelife.Alive
```

Comment:
corelife does not have an explicit invalid value, but it's `Validate() error` method deals out an error when the value is _not_ in `{alive, dying, dead}`, so shouldn't this just return an empty string `corelife.Value`.

Reply:
returned "" instead